File: Implementações/C/Fibonacci.c
#include <stdio.h>

int numero;
int resultado;

//Função que calcula e imprime a sequência de Fibonnacci:

int Fibonacci(int quantidade) {
	int resultado, primeiroNumero, segundoNumero;

	primeiroNumero = 0;
	segundoNumero = 1;

	for (int i = 0; i < quantidade; ++i)
	{
		resultado = primeiroNumero + segundoNumero;
		segundoNumero = primeiroNumero;
		primeiroNumero = resultado;
		printf("%i ", resultado);
	}
	return resultado;

}


int main() {
	puts("Digite um numero para a serie de Fibonacci:");
	scanf("%2i", &numero);
	printf("A sequencia para os %i primeiros numeros de Fibonacci é: \n", numero);
	Fibonacci(numero);

	return 0;

}

File: Implementações/Python/Fibonacci.py

#Essa função encontra e improme a sequência até determinado número:
def fibonacci(quantidade):
    primeiroNum, segundoNum = 0, 1
    
    while primeiroNum < quantidade:
        print(primeiroNum, end=' ')
        primeiroNum, segundoNum = segundoNum, primeiroNum + segundoNum
    print()

#Essa outra impime a quantidade de digitos selicionados:

def fibonacci2(quantidade):
    primeiroNum, segundoNum = 0, 1

    for i in range(quantidade):
        print(primeiroNum, end=' ')
        primeiroNum, segundoNum = segundoNum, primeiroNum + segundoNum
        pass
    print()


numeroFibo = input("Digite um número para a série de Fibonacci:\n")      
numeroFibo = int(numeroFibo)

print(f"\nA sequência de Fibonacci até o numero {numeroFibo} é: ")
fibonacci(numeroFibo)

print(f"\nA sequência de Fibonacci para os primeiros {numeroFibo} números é: ")
fibonacci2(numeroFibo)